### PR TITLE
Update apollo-server-express and lodash

### PR DESCRIPTION
**Pull request by german-glob**
Updated apollo-server-express and lodash to the newest versions.

I'm going to wait with merging this till https://github.com/globality-corp/nodule-graphql/pull/125 will be completed.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:8.11.3
+    - image: circleci/node:10.23.1
```
Comment (german-glob) [inline .circleci/config.yml:4]: Bumped node, because one of the dependencies of `apollo-server-express` - `lru-cache 6.0.0` is incompatible with node 8.

Comment (thill-glo) [reply .circleci/config.yml:4]: This breaks the spooky hash dependency. Please see: https://globalityinc.slack.com/archives/C0Q4GRU4T/p1610736515014200 and https://globalityinc.slack.com/archives/CP680HJ9E/p1611049553001700

**Review by thill-glo — CHANGES_REQUESTED** [review]

**Review by thill-glo — APPROVED** [review]

**Comment by thill-glo** [comment]
Sorry @german-glob didn't spot you were already waiting on @simpleigh-globality's PR. All good!

**Comment by simpleigh-globality** [comment]
> Sorry @german-glob didn't spot you were already waiting on @simpleigh-globality's PR. All good!

Yep, sorry. I had a play with this last night but ran into some frustrating issues where `process.versions.napi` reports a different number than the value of `NAPI_VERSION` in the header files that `node-gyp` uses to compile the extension. I got the library running on an older version of Node but need to make the unit tests conditional on the version...

**Comment by simpleigh-globality** [comment]
Replaced by #128

**Pull request closed without merging**